MonoGame.Framework/Desktop/OpenTKGameWindow.cs

```diff
@@ -286,24 +270,11 @@ private void UpdateWindowState()
                 else
                     window.WindowState = windowState; // usually fullscreen-stuff is set from the code
                 
-                // fixes issue on linux (and windows?) that AllowUserResizing is not set any more when exiting fullscreen mode
-                WindowBorder desired;
-                if (_isBorderless)
-                    desired = WindowBorder.Hidden;
-                else
-#if LINUX
-                    desired = WindowBorder.Resizable;
-#else
-                    desired = _isResizable ? WindowBorder.Resizable : WindowBorder.Fixed;
-#endif
-                if (desired != window.WindowBorder && window.WindowState != WindowState.Fullscreen)
-                    window.WindowBorder = desired;
+                updateborder = 2;
```

Comment:
Sure I'll add it, one sec.

The reason why it's 2 is we need a very small delay between resizing the window and setting the borders to avoid Linux bug.